test/integ_tests/test_batch.py

```diff
@@ -44,7 +44,11 @@ def func(weights):
     qnode_aws = qml.QNode(func, dev_aws)
     qnode_default = qml.QNode(func, dev_default)
 
-    weights = qml.init.strong_ent_layers_uniform(layers, qubits)
+    if qml.version() >= "0.20.0":
+        shape = qml.templates.StronglyEntanglingLayers.shape(layers, qubits)
+        weights = np.random.random(shape)
+    else:
```

Comment:
No need for this if else. Just use the 0.20 version.

Reply:
Oh okay, I thought when using <PL-0.20 `qml.init.strong_ent_layers_uniform` module will be present and otherwise. Thanks!